handlers/playback.go

```diff
@@ -23,12 +23,20 @@ func PlaybackHandler() httprouter.Handle {
 			return
 		}
 
+		gatingParamName := "accessKey"
+		gatingParam := req.URL.Query().Get(gatingParamName)
+		if gatingParam == "" {
+			gatingParamName = "jwt"
+			gatingParam = req.URL.Query().Get(gatingParamName)
+		}
+
 		playbackReq := playback.Request{
-			RequestID:  requestID,
-			PlaybackID: params.ByName("playbackID"),
-			File:       params.ByName("file"),
-			AccessKey:  req.URL.Query().Get(playback.KeyParam),
-			Range:      req.Header.Get("range"),
+			RequestID:       requestID,
+			PlaybackID:      params.ByName("playbackID"),
+			File:            params.ByName("file"),
+			GatingParam:     gatingParam,
```

Comment:
Interesting! So the current logic already detects if it's a jwt and handles as that? I thought it got the "type" somewhere, but it makes sense that it doesn't